packages/backend/src/tokens/utils.ts

```diff
@@ -0,0 +1,11 @@
+export function toSSRResource(resource?: { private_metadata: any } | { privateMetadata: any }) {
```

Comment:
I wonder if this should be a standalone helper like this or part of the resources themselves eg `toClientSideJson` or something similar

Reply:
I'd say standalone as we don't have proper resources in @clerk/backend core. There are more DTOs rather than models.